internal/config/config.go

```diff
@@ -192,6 +195,18 @@ func parseHoldTime(ht string) (time.Duration, error) {
 	return rounded, nil
 }
 
+func parseKeepaliveTime(ka string) (time.Duration, error) {
+	if ka == "" {
```

Comment:
Can the default logic be the same as the current stack (you mentioned it being 1/3 of holdtime)?

Reply:
current stack doesn't configure it and the default is not configured is 30sec which is 1/3 of the default holdtime

Reply:
> current stack doesn't configure it and the default is not configured is 30sec which is 1/3 of the default holdtime

so it's always 30 sec or it's 1/3 of the holdtime?

Reply:
the default holdtime is 90s hence default ka is 30s if u changed the holdtime u need to change the ka value and at this point it does't have to be 1/3 it can be anything based on what you are peering with

Reply:
Yes, but if the user specifies something different as HoldTime, the keep alive time is gonna always be 1/3 (https://github.com/metallb/metallb/blob/9cb1653a87673ec91971a60c873204e0752ff97b/internal/bgp/native/native.go#L363). 
So, to preserve the current behaviour, we should do the same. If a user specifies the HoldTime but not the keep alive time, we should use HT/3

Reply:
at the description of this PR I said metallb it assumes its always 1/3 hence its not a config options this is not typical for other stacks they export it both timers to config with only one condition ka timer can't be less than ht

Reply:
Not sure I am missing something. 
What I am saying here is, if a user specifies an HT but not a keep alive, let's use HT/3 as default instead of fixing it to 30 seconds, so one configuration that is working today will continue working when switching implementation.

Reply:
`neighbor PEER timers (0-65535) (0-65535)`
they are configured together so you either keep default for both or change both

Reply:
They are configured together in FRR, not in MetalLB.
With the code of this PR, if a user sets heartbeat but not keepalive, keepalive is defaulted to 30 seconds.

We either force them to be configured both in MetalLB, or we use the same defaulting mechanism we had in place before, calculating keepalive to be heartbeat/3
I don't think it's fair to force it in metalLB since it would break the compatibility promise.

Reply:
current stack completly ignore the keepalive settings this field in the config isn't user or referrenced anywhere in the metallb stack so the code will stll under the hood change ka to 1/3 of the new HT